documentation/api.md

```diff
@@ -56,8 +56,7 @@ The user facing methods of `FungibleToken` are
 @method async transfer(from: PublicKey, to: PublicKey, amount: UInt64)
 @method async approveBase(updates: AccountUpdateForest): Promise<void>
 @method.returns(UInt64) async getBalanceOf(address: PublicKey): Promise<UInt64>
-@method.returns(UInt64) async getCirculating(): Promise<UInt64>
-@method async updateCirculating()
+@method.returns(UInt64) async getCirculating(updateContractState: Bool): Promise<UInt64>
```

Comment:
just a note, we wouldn't _have_ to change the public API just because we're consolidating `@method`s and reduce code duplication (both of which are a good thing)

we could also add wrappers that are not `@method`s which call the base method internally, e.g. `updateCirculating()` could call `getCirculating(Bool(true))`

the token contract does the same thing with `transfer()` and `approveAccountUpdate()` which call the same `@method approveBase()` internally

Reply:
Good idea, let me do that!